openfisca_web_site/templates/france/fr/tree/elements/index.py

```diff
@@ -112,8 +112,8 @@ def iter_items(self):
             country = [u'france'],
             description = u"Estimez le coût d'une embauche dans votre entreprise, et combien le salarié touchera.",  # noqa
             logo_url = urls.get_url(ctx, self.url_path, 'images', 'logo-openfisca.png'),
-            owner = u"SGMAP",
-            source_url = u'http://embauche.sgmap.fr/',
+            owner = u"DINSIC",
+            source_url = u'http://embauche.beta.gouv.fr/',
```

Comment:
*embauche* doesn't use openfisca anymore but I guess such a discussion should take place in another PR